### DISCUSSION TITLE
Stresses and forces in training data

### DISCUSSION THREAD

**Discussion by apoletayev** [mir-group/allegro#12] [Q&A] [answered]
My question stems from trying to train allegro models on AIMD simulation data taken in NPT and NVT ensembles and processed from VASP to extxyz. Do stresses from an NPT simulation contribute to forces on atoms for the purpose of training? What would be the appropriate way of including / excluding them for training?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @apoletayev ,

Thanks for your interest in our code! Not sure what you mean by "contribute to forces on atoms for the purpose of training"... 

We support training on stress/virial labels and the approach is the same as for a NequIP model using the same options in the `nequip` framework; see `minimal_stress.yaml` in `nequip` and various threads, like this one https://github.com/mir-group/nequip/issues/114, in the NequIP discussions and issues.

**Reply by apoletayev** [reply]
Thank you! After more testing, I found that the problem with my code was more that I had very similar-looking frames that differed by spin in a spin-polarized DFT calculation, and the training was understandably thrown off by that.
